css/index.css

```diff
@@ -143,4 +208,43 @@ footer nav {
 footer nav a {
     color: white;
     text-decoration: none;
+}
+
```

Comment:
Notice how at least 2 of these atiles and btiles are identical?  Here you can just combine them to use the same class or put the class names together like .atiles .btiles { \* css here *\ }.  the about-top-text also is almost the same.  You can still combine that with .atiles .btiles .about-top-text { \* css here *\ } for everything that is the same then do .about-top-text { *\ css that is different *\}.  This would save 3 - 4 lines of extra CSS.